2023-Season/src/main/java/frc/robot/Robot.java

```diff
@@ -107,12 +107,23 @@ public void simulationPeriodic() {
   }
 
   private void teleopDrive() {
-    var translationX = OneDimensionalLookup.interpLinear(Constants.XY_Axis_inputBreakpoints,
-        Constants.XY_Axis_outputTable, _driverController.getLeftY()) * Constants.MAX_VELOCITY_METERS_PER_SECOND;
-    var translationY = OneDimensionalLookup.interpLinear(Constants.XY_Axis_inputBreakpoints,
-        Constants.XY_Axis_outputTable, _driverController.getLeftX()) * Constants.MAX_VELOCITY_METERS_PER_SECOND;
-    var rotationZ = OneDimensionalLookup.interpLinear(Constants.RotAxis_inputBreakpoints, Constants.RotAxis_outputTable,
-        _driverController.getRightX()) * Constants.MAX_ANGULAR_VELOCITY_PER_SECOND;
+    var translationX = OneDimensionalLookup.interpLinear(
+      Constants.XY_Axis_inputBreakpoints,
+      Constants.XY_Axis_outputTable, 
+      _driverController.getLeftY()
+    ) * Constants.MAX_VELOCITY_METERS_PER_SECOND;
```

Comment:
I would argue this line is harder to read than before, you can move the math to be on the same line and keep each parameter separate.

Reply:
got you will move the math stuff on the same line